client/my-sites/plugins/controller.js

```diff
@@ -186,16 +186,10 @@ function renderProvisionPlugins( context ) {
 }
 
 const controller = {
-	plugins( context, next ) {
+	plugins( context ) {
```

Comment:
Isn't it a problem that `next` is not called here? If you later add `makeLayout` and `render` handlers, they won't be executed. Also, other routes that match the path and are registered later than this one won't be executed.

Reply:
I think it's fine to leave it out for now to keep this function in align with the rest of the controller's middlewares (i.e. rendering is the last step and none of the rendering middlewares currently calls `next()`). I removed it because I removed the logic that was calling it initially and I didn't want to add it to the end _just yet_.

As a next step in another PR I'll want to add `next` to _all_ these middlewares. Here I'm just preparing for it.